### PR TITLE
Pop up project

### PR DESCRIPTION
**Pull request by Ashnaali3255**
**Description**
For this milestone of your portfolio website, you will implement a popup window that includes the project details in both desktop and mobile. In order to do that, you will need to store the information about your projects in a JavaScript object.

IMPORTANT NOTE: Read all requirements before you start building your project.

**General requirements**
Make sure that there are [no linter errors](https://github.com/microverseinc/linters-config).
Make sure that you used correct [GitHub flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md).
Make sure that you documented your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).
HTML/CSS & JavaScript requirements
Follow our list of [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).
Follow our list of [best practices for JavaScript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md).
Project requirements
IMPORTANT NOTE: For each Microverse project, you need to create a separate pull request. [Read how to work with multiple pull requests in your one repository in the most efficient way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/multiple_pull_requests.md).

You should implement the following interactions:

When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
When the user clicks (or taps) the close (X) button, the popup disappears.
In order to associate each project with the popup window details, you must refactor the project section:

You need to use a JavaScript array to store all of the information for all projects.
For each project you need to store the following pieces of data in a JavaScript object, at a minimum: name, description, featured image, technologies, link to live version, link to source.
You need to update the main page so that the projects section is created dynamically using the information stored in that JavaScript object. Remember, all of the HTML in that section is created when the page loads.
After that, you could implement the popup window.
You should implement popups for both mobile and desktop screen sizes:

Mobile popup
Mobile popup

Desktop popup
Desktop popup

We included a screenshot of Template 5 for reference, but you should follow the template that you chose.

You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

### CODE REVIEW

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
Please check the review below.

File: style.css [hunk 1]
```diff
@@ -472,6 +472,138 @@ hr {
...
+}
+
+.model_header {
+  display: flex;
+  height: 20px;
+  width: 100%;
+  justify-content: space-between;
+  align-items: center;
+  margin: 12px 0;
+}
```
Comment (omar-muhamad) [inline style.css]: - [ ] The `see source` button is overflowing from the page bottom, Please fix this button positioning.

| Given Design  | Your Design |
| ------------- | ------------- |
| image  | image |

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi Team,

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used the correct workflow.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: style.css [hunk 2]
```diff
@@ -472,6 +472,142 @@ hr {
...
+  height: calc(100vh - 20px);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: #fff;
+  border-radius: 8px;
+  padding-bottom: 10px;
+  padding-top: 25px;
+  margin: 12px auto;
+}
```
Comment (TadesseDev) [inline style.css:487]: - [Optioanl] (`HIGHLY RECOMMENDED`). Good job addressing the changes previously requested, But I notice on a mobile simulation your modal overflows and there is no possibility to scroll up. This is a bad thing when it comes to UX. Please consider addressing this.

Screenshot 2023-02-01 105523

**Review by TadesseDev — APPROVED** [review]
### Hi @Ashnaali3255 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
barly


## Highlights
 - [ ] Good job making the changes requested by the previous code reviewer. ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**